Build fails with: ERROR: "mcount" [/home/kees/src/rust-linux/hello-world/helloworld.ko] undefined!

This is likely some kind of mismatch between gcc and clang when dealing with ftrace. Disabling CONFIG_FTRACE didn't help, though. :(

```
$ KDIR=~/src/linux make
make -C /home/kees/src/linux M=/home/kees/src/rust-linux/hello-world
make[1]: Entering directory '/home/kees/src/linux'
  LD [M]  /home/kees/src/rust-linux/hello-world/helloworld.o
  Building modules, stage 2.
  MODPOST 1 modules
WARNING: modpost: missing MODULE_LICENSE() in /home/kees/src/rust-linux/hello-world/helloworld.o
see include/linux/module.h for more information
ERROR: "mcount" [/home/kees/src/rust-linux/hello-world/helloworld.ko] undefined!
scripts/Makefile.modpost:103: recipe for target 'modules-modpost' failed
make[2]: *** [modules-modpost] Error 1
Makefile:1627: recipe for target 'modules' failed
make[1]: *** [modules] Error 2
make[1]: Leaving directory '/home/kees/src/linux'
Makefile:8: recipe for target 'all' failed
make: *** [all] Error 2

```

--- a/kernel-cflags-finder/Makefile
+++ b/kernel-cflags-finder/Makefile
@@ -1,7 +1,6 @@
 ifneq ($(KERNELRELEASE),)
 obj-m += dummy.o
 clean-files := dummy.c
-ccflags-y += -Wno-unknown-warning-option
 
 # Some systems for installing kernel headers (e.g. Debian's) happen to
 # trigger the out-of-tree build code because the kernel headers directly
@@ -30,7 +29,7 @@ else
 KDIR ?= /lib/modules/$(shell uname -r)/build
 CLANG ?= clang
 all:
-	$(MAKE) -C $(KDIR) M=$(CURDIR) CC=$(CLANG)
+	$(MAKE) -C $(KDIR) M=$(CURDIR) CC=$(CLANG) CONFIG_CC_IS_CLANG=y
 clean:
 	$(MAKE) -C $(KDIR) M=$(CURDIR) clean
 endif

